kolibri/core/assets/src/views/k-select/index.vue

```diff
@@ -142,7 +142,6 @@
   .k-select-inline
     display: inline-block
     vertical-align: bottom
-    width: 100px
```

Comment:
The last thing would be to keep this width since we already added `width: auto` in the language switcher.

Reply:
Done adding it @christianmemije.